tests/e2e/test_prices.py

```diff
@@ -30,11 +35,6 @@ def test_eth_in_token(self):
             3032.45 / 0.435229,
             delta=0.0001,
         )
-        self.assertAlmostEqual(
```

Comment:
I actually like this test. I would not remove it, It shows that its also working with 6 digit tokens

Reply:
Yea. The thing is we have no need for USDC and then the token enum would have a useless token in it.

Reply:
But I can put it back anyway.